app/components/summary_card/view.rb

```diff
@@ -0,0 +1,40 @@
+class SummaryCard::View < ViewComponent::Base
+  # validates :rows, presence: true
```

Comment:
is this commented out code needed?

Reply:
Refactored the component to only include what we have and need currently